docat/docat/app.py

```diff
@@ -113,6 +113,44 @@ def get_project(project):
     )
 
 
+@app.post("/api/{project}/icon", response_model=ApiResponse, status_code=status.HTTP_200_OK)
+@app.post("/api/{project}/icon/", response_model=ApiResponse, status_code=status.HTTP_200_OK)
+def upload_icon(
+    project: str,
+    response: Response,
+    file: UploadFile = File(...),
+    docat_api_key: Optional[str] = Header(None),
+    db: TinyDB = Depends(get_db),
+):
+    project_base_path = DOCAT_UPLOAD_FOLDER / project
+    icon_path = project_base_path / "logo.png"
+
+    if not project_base_path.exists():
+        response.status_code = status.HTTP_404_NOT_FOUND
+        return ApiResponse(message=f"Project {project} not found")
+
+    if file.content_type != "image/png":
```

Comment:
Can't we just check for any type of image? `file.content_type.startswith("image/")` and then save the icon without extention? just `project_base_path / "logo"` The browser will figure out what to display :)

Reply:
Good idea, I didn't think of that. :+1: